configure.ac

```diff
@@ -256,12 +256,10 @@ dnl Check for libtiff
 dnl ---------------------------------------------------------------------------
 
 AC_ARG_ENABLE([tiff],
-              AS_HELP_STRING([--disable-tiff],
-                    [Disable TIFF support. Strongly discouraged !]),
-              [enable_tiff=no],
-              [enable_tiff=yes])
+              AS_HELP_STRING([--enable-tiff],
+                    [Enable TIFF support; strongly encouraged to read some grids [default=yes]]))
```

Comment:
Before was incorrect, too. The third argument to `AC_ARG_ENABLE` is  action-if-provided, and the fourth argument is action-if-not-provided. That's not what it used to do. It's possible there are more of these in `configure.ac`.

However, there's not much benefit to changing the help string if the default is enabled. `AC_ARG_ENABLE` defines _both_ `--enable-tiff` and `--disable-tiff` (and also accepts `--enable-tiff=no`, too).

Reply:
@QuLogic so is this [`AC_ARG_ENABLE`](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Package-Options.html) good? The third and fourth arguments are optional. And I've stress tested all the combinations to ensure that I see (or don't) a line "checking for TIFF... yes".

The only other example using `AC_ARG_ENABLE` is for `--enable-lto`, which is done similar to this.